### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by magiepooh**

### CODE REVIEW

File: app/src/main/java/com/kamron/pogoiv/MainActivity.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ protected void onCreate(Bundle savedInstanceState) {
         // Set up Crashlytics, disabled for debug builds
         Crashlytics crashlyticsKit = new Crashlytics.Builder()
                 .core(new CrashlyticsCore.Builder()
-                .disabled(BuildConfig.DEBUG).build())
+                        .disabled(BuildConfig.DEBUG).build())
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: Nitpick: FWIW, this reindentation does not look intentional.

Comment (sarav) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: Does look right. He's indenting it to line up with the new inner object.

Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: Oh sorry, I misread the parens!

**Comment by sarav** [comment]
Hey, due to some mistake of mine the master branch had to be rewound in history. You might have to rebase or cherry-pick your changes and push you branch again and make sure it only contains your changes and not old crap from me.

**Comment by Blaisorblade** [comment]
So I'm volunteering to update the refactorings to all the other changes, I hope that's OK, sorry for not saying earlier.

I'm not pushing the reformatting though for now, though it'd be good—too easy to get conflicts.

**Comment by Blaisorblade** [comment]
The second commit is in #112.

**Comment by Blaisorblade** [comment]
#112 merged... only the reformatting is left here.

**Comment by Blaisorblade** [comment]
We'll do the reformatting from scratch. Closing. Thanks again @magiepooh!

**Comment by magiepooh** [comment]
@Blaisorblade Thanks!!

**Pull request closed without merging**